### ISSUE TITLE
Podenv build fails on `import Relude.Extra.Lens (Lens')`

### ISSUE THREAD

**Issue by JayDoubleu** [podenv/podenv#88]
Podenv build fails on `import Relude.Extra.Lens (Lens')`
Not sure if i'm building the same way as last time, using https://github.com/podenv/podenv/blob/main/docs/howtos/contribute.md .

```cabal
Building     dhall-1.40.1 (lib)
Installing   dhall-1.40.1 (lib)
Completed    dhall-1.40.1 (lib)
Configuring library for podenv-0.1.0.0..
Preprocessing library for podenv-0.1.0.0..
Building library for podenv-0.1.0.0..
[ 1 of 11] Compiling Paths_podenv
[ 2 of 11] Compiling Podenv.Dhall
[ 3 of 11] Compiling Podenv.Prelude

src/Podenv/Prelude.hs:49:1: error:
    Could not find module ‘Relude.Extra.Lens’
    Perhaps you meant
      Relude.Extra.Enum (from relude-0.2.0)
      Relude.Extra.Map (from relude-0.2.0)
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
49 | import Relude.Extra.Lens (Lens')
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

<no location info>: warning: [-Wunused-packages]
    The following packages were specified via -package or -package-id flags,
    but were not needed for compilation:
      - typed-process-0.2.7.0
      - th-env-0.1.0.3
      - optparse-applicative-0.16.1.0
      - linux-capabilities-0.1.0.0
      - gitrev-1.3.1
      - SHA-1.6.4.4
cabal: Failed to build podenv-0.1.0.0 (which is required by exe:podenv from
podenv-0.1.0.0).

```

**Comment by juhp** [comment]
How are you building? I think your relude looks too old.
Maybe a lowerbound should be added.

**Comment by TristanCacqueray** [comment]
Thanks for the report and feedback. Could you please try #89 and see if that fixes your issue?

**Comment by JayDoubleu** [comment]
Awesome. Thanks for #89 just tested it and now builds properly.

@juhp tried building using freshly created F35 and F34 toolbox containers.

**Comment by TristanCacqueray** [comment]
I haven't check which cabal-install comes in fedora, but perhaps it is too old to work without min-bound. I guess relude was the only failure because the other dependencies are shared with the dhall library which enforce version bounds.

**Comment by juhp** [comment]
@JayDoubleu btw I have a [copr repo](copr.fedorainfracloud.org/coprs/petersen/podenv/)
I see - still curious how cabal choose such an old relude for you: eg Fedora had 0.7.0.
Maybe you had it installed somehow.

**Comment by JayDoubleu** [comment]
Actually a grabbed the binary from your copr while was waiting on this fix :)
I'm on Silverblue and trying my best to survive without layered packages AKA default terminal just launches GUI terminator from within toolbox and podenv is going to server as other way to get GUI and isolated apps where I can't or dont want to use flatpaks from flathub.

**Comment by juhp** [comment]
Cool, I am also a silverblue user.

Ya it has also occurred to me that it would not hurt to make the executable build artifact available (even though it is built on Ubuntu).

**Comment by TristanCacqueray** [comment]
Thank you again for the feedback. So I've created https://github.com/podenv/podenv/milestone/1 to track the 1.0 release progress.
